index.yaml

```diff
@@ -3520,171 +3520,6 @@ entries:
     urls:
     - assets/rancher-node-exporter/rancher-node-exporter-1.16.201.tgz
     version: 1.16.201
-  rancher-operator:
```

Comment:
As we know we are deleting this whole section, can you just delete it manually instead of running `make charts` to avoid all those timestamp changes?

Reply:
good idea, done